packages/core-mobile/app/contexts/SendTokenContext.tsx

```diff
@@ -164,13 +198,15 @@ export const SendTokenContextProvider = ({
         })
         .catch(reason => {
           setSendStatus('Fail')
-          showTransactionErrorToast({
-            message: getJsonRpcErrorMessage(reason)
-          })
-          AnalyticsService.capture('SendTransactionFailed', {
-            errorMessage: reason.message,
-            chainId: activeNetwork.chainId
-          })
+          if (!isUserRejectedError(reason)) {
+            showTransactionErrorToast({
+              message: getJsonRpcErrorMessage(reason)
+            })
+            AnalyticsService.capture('SendTransactionFailed', {
+              errorMessage: reason.message,
+              chainId: activeNetwork.chainId
+            })
+          }
```

Comment:
we were reporting and showing toast even for the user rejected error. this fixes it.